dao/src/main/java/cz.muni.fi.pa165.soccermanager/entity/Match.java

```diff
@@ -1,4 +1,112 @@
 package cz.muni.fi.pa165.soccermanager.entity;
 
+import javax.persistence.*;
+import java.util.Date;
+
 public class Match {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    //homeTeam, awayTeam, place, date, homeTeamGoalsScored, awayTeamGoalsScored
+
+    @Temporal(TemporalType.DATE)
+    private Date date;
+    private String stadium; // TODO: define enum of stadiums
+    private Team homeTeam;
```

Comment:
team should have ManyToOne JPA notation

Reply:
You should also add JoinTable notation, sorry forgot to mention this before. Check my code if you need help with it